spec/requests/emojis_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Emojis' do
+  describe 'GET /emojis/:id' do
+    let(:emoji) { Fabricate(:custom_emoji, shortcode: 'coolcat') }
+
+    it 'returns http success with correct json' do
+      get emoji_path(emoji, format: :json)
```

Comment:
What about the following to ensure consistency with the description?
```suggestion
      get "/emojis/#{emoji.id}"
```